client/src/components/Artist/ArtistTrackGroup.tsx

```diff
@@ -98,16 +100,13 @@ const ArtistTrackGroup: React.FC<{
                           color: var(--mi-light-foreground-color);
                           font-style: italic;
                         `
+                      + " track-group-name"
                   }
                 >
                   {trackGroup.title.length ? trackGroup.title : t("untitled")}
                 </Link>
               )}
-              {trackGroup.artist && (
-                <Link to={getArtistUrl(trackGroup.artist)}>
-                  {trackGroup.artist?.name}
-                </Link>
-              )}
+              {<ArtistName artist={trackGroup.artist} />}
```

Comment:
Do you have [Prettier](https://prettier.io/) enabled for this project? This should automatically turn this into: 

```suggestion
              <ArtistName artist={trackGroup.artist} />
```

decluttering it a little bit. It's an outstanding project to add it to the repo by detault